x-pack/plugins/cases/public/components/connector_selector/form.tsx

```diff
@@ -49,6 +50,14 @@ export const ConnectorSelector = ({
     connectors.find((connector) => connector.id === field.value)
   );
 
+  const ADD_CONNECTOR_HELPER_TEXT = i18n.translate(
```

Comment:
nit: I'd move this outside the definition of `ConnectorSelector`, something like line 18

Reply:
moved it, thanks :)